.docker/ci/Dockerfile

```diff
@@ -17,10 +17,10 @@ RUN \
     cd src && \
     #
     # Download moveit source, so that we can get necessary dependencies
-    wstool init --shallow . https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall && \
+    wstool init --shallow . https://raw.githubusercontent.com/ros-planning/moveit/master/moveit.rosinstall && \
     #
     # Update apt package list as previous containers clear the cache
-    apt-get -qq update && \
+    apt-get -qq update --fix-missing && \
```

Comment:
Why do you think, it's valuable to ignore missing packages?
Shouldn't we get noticed about this issue?

Reply:
I was going by the console output from a failed docker build which recommended --fix-missing. I didn't realize that by fix they actually meant ignore haha. I have reverted this change